builds/installing-openshift-builds.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="installing-openshift-builds"]
+= Installing {builds-shortname}
+:context: installing-openshift-builds
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+
+As a cluster administrator, you can install {builds-title} on an {product-title} cluster. 
```

Comment:
@shipsing from where we are fetching the value for `product-title`.
On the preview page, I can see the sentence as:
```
As a cluster administrator, you can install Red Hat OpenShift Builds on an **Red Hat OpenShift Builds cluster**.
```

Whereas, I think the following might be more appropriate.
```
As a cluster administrator, you can install Red Hat OpenShift Builds on an **Red Hat OpenShift cluster**.
```
Please let me know in case there have been a previous discussion around it, and team has already agreed to proceed the former way.

Reply:
@apoorvajagtap I need to update the atribute with {ocp-product-title} and then you would see `As a cluster administrator, you can install Red Hat OpenShift Builds on an OpenShift Container Platform cluster`. i need to use the correct attribute.